gen_png.py

```diff
@@ -145,17 +159,24 @@ def main():
             elif 'D' == split_file[0]:
                 deleted.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
             elif re.match(r"R[0-9]+", split_file[0]):
-                deleted.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
-                modifications.append(split_file[2])
+                renamed_old.append(split_file[1].removeprefix('svg/').removesuffix('.svg'))
+                renamed_new.append(split_file[2])
```

Comment:
Why are the svg parts removed from the old, but not the new?

Reply:
The delete method depended on the content being removed preliminary.
I moved the cleanup into the method.
This makes things cleaner and more robust. Thanks!